tools/integration-test-baseline.php

```diff
@@ -33,7 +34,9 @@ function copyTests(string $directory): void
             $tempFile = $tempDir . '/' . $fileName;
             $outputFile = $expectedDir . '/' . $fileName;
             if (file_exists($tempFile)) {
-                if (pathinfo($tempFile, PATHINFO_EXTENSION) === 'html') {
+                if (pathinfo($tempFile, PATHINFO_EXTENSION) === 'log') {
+                    echo sprintf("Ignoring log file %s\n", $outputFile);
```

Comment:
```suggestion
                    echo sprintf("Ignoring log file %s\n", $outputFile);
                    continue;
```

I would like to see early `continue` also for the other cases to reduce cognitive complexity.

Reply:
I just don't see the sense of using continue (which I find cognitive complex) when you can easily do without it?

Reply:
done, please have another look